contract-map.json

```diff
@@ -1,4 +1,16 @@
 {
+  "0xc3cB5D41C2f79C2F4f33A5D8a6DE72D0D87b1DB3": {
```

Comment:
Can you provide a citation for this address? I will need to close this PR if not.